src/roaring.c

```diff
@@ -129,6 +129,29 @@ void roaring_bitmap_add_many(roaring_bitmap_t *r, size_t n_args,
     }
 }
 
+void roaring_bitmap_contains_multi(roaring_bitmap_t *r, size_t n_args,
+                             const uint32_t *vals, bool *results) {
+    if (n_args == 0) return;
```

Comment:
I'd rather have the `return` in its own line, some may miss it.

Reply:
And even then, the loop guard already makes sure nothing really interesting happens if we remove the `memset`. The fixed overhead of setting a few `0`s isn't likely to make a big difference.

Reply:
fixed

Reply:
You do not have to change it, but I prefer...

```C++
if (n_args == 0) { return; }
```